### DISCUSSION TITLE
Is there a way to revert the `@name` template to contain the "almost" iso-8601 time format?

### DISCUSSION THREAD

**Discussion by stdedos** [openstyles/stylus#1756] [Q&A]
Is there a way to revert the `@name` template to contain the "almost" iso-8601 time format?

**Comment by tophf** [comment]
It's not implemented.

**Reply by stdedos** [reply]
Was it planned for implementation / Is there a Github issue for it?

https://github.com/openstyles/stylus/issues?q=is%3Aissue+name+template+is%3Aopen does not seem to have it

**Reply by tophf** [reply]
It's not planned. Do you have an idea how it can be done conceptually?

**Reply by stdedos** [reply]
js/ts is not my cup of tea 😅

If it were Python, and we were talking of a "simple" string interpolation, I'd say that `"@name     %(page_domain)s - %(user)" % params` would be nice - like argparse formats its `help=` text (the key values of the argument can be re-used in `help=`).

However, how would that work for complex arguments (like a customly-formated date), or in js - I have no idea 😓 

Ofc I could pretend that you could offer `date`, `localeDate`, `iso8601date`, etc - but I know that that doesn't scale 😕

**Reply by tophf** [reply]
I can implement it the Violentmonkey way as `{{date:YYYY-MM-DD}}` using a subset of popular MomentJS tokens.
